# ArrowDetection2D.msg
Header header
string label
float32 confidence
int32 x
int32 y
